run_dir/static/js/project_samples.js

```diff
@@ -354,6 +354,13 @@ function load_all_udfs(){
         safeobj(key).html(auto_format(value));
       }
 
+      // highlight if project is shared
+      else if (prettify(key) == 'shared') {
+        if (value != 'No') {
+          $('#type').append('<span class="label label-warning">Shared</span>');
```

Comment:
Ok, I don't really understand but I think it's difficult without seeing it on the page..